src/components/visualizations/chart/chartOptionsBuilder.ts

```diff
@@ -369,6 +369,7 @@ export function getHeatmapSeries(
                     borderWidth: 0,
                     pointPadding: 2,
                     color: nullColor,
+                    ignoredInDrillEventContext: true,
```

Comment:
Property `ignoredInDrillEventContext` is used internally and not is not used by Highchart. Please add comment to explain it.

Reply:
something like :)
`// act as a flag to check and remove this null-value point in drill message`

Reply:
updated. Thanks :)